events/hfe_events_init.txt

```diff
@@ -1055,9 +1108,18 @@ country_event = {
 					initializer = hfe_initializer_future_main
 				}
 				set_global_flag = hfe_future_spawned
+				if = {
+					limit = { has_global_flag = HFE_TSP_spawn_test }
+					log = "Kad Republic not found but clay spawned"
+				}
 			}
 		}
 		event_target:hfe_future_country = {
+			hfe_effect_init_give_tech_t5 = yes
+			add_modifier = {
```

Comment:
What is this modifier

Reply:
It gives the Kad Republic their full technology.

Reply:
It's the official Vanilla rendition Modifier that is assigned to all Fallen Empires. It mainly deals with Fleet Limits and some upkeep that would send them into a death spiral.

Reply:
I didnt notice any death spiral

Reply:
There isn't, but they were over one hundred seventy ships above their Fleet Limit. This means that when they awake, they will face both deficits and the inability of the AI to build more ships. That is why Paradox gave Fallen Empires that Modifier in the first place.